stingray/base.py

```diff
@@ -2134,6 +2134,147 @@ def sort(self, reverse=False, inplace=False):
             mask = mask[::-1]
         return self.apply_mask(mask, inplace=inplace)
 
+    def fill_bad_time_intervals(
+        self,
+        max_length=None,
+        attrs_to_randomize=None,
+        buffer_size=100,
+        uniform=None,
+        seed=None,
+    ):
+        """Fill short bad time intervals with random data.
+
+        To fill the gaps in all but the time points (i.e., flux measures, energies), we take the
+        ``buffer_size`` (default 100) valid data points closest to the gap and repeat them randomly
+        with the same empirical statistical distribution. That is, if the "blabla" attributes, in
```

Comment:
What's a `blabla` attribute? :D

Reply:
Just a random name. It means something like "blabber" in Italian, we use it as an informal "etc. etc."

Reply:
I changed it to `my_fancy_attr`